src/Libraries/Microsoft.AspNetCore.HeaderParsing/Microsoft.AspNetCore.HeaderParsing.csproj

```diff
@@ -10,6 +10,7 @@
     <TargetFrameworks>$(NetCoreTargetFrameworks)</TargetFrameworks>
     <!-- using the ConfigurationBinder source generator is blocked by https://github.com/dotnet/runtime/issues/94547, which was only fixed in 9.0+ -->
     <EnableConfigurationBindingGenerator Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net9.0'))">true</EnableConfigurationBindingGenerator>
+    <InterceptorsNamespaces Condition="'$(EnableConfigurationBindingGenerator)' == 'true'">$(InterceptorsNamespaces);Microsoft.Extensions.Configuration.Binder.SourceGeneration</InterceptorsNamespaces>
```

Comment:
Need to figure out if this is the correct fix here. I'm not sure why this isn't enabled automatically when setting `EnableConfigurationBindingGenerator=true`

Reply:
I've figured this out. Previously in .NET 9 SDK, we would use the NuGet package to reference/build, which has these settings inside. But in .NET 10 the NuGet package no longer gets used, so no one is setting this property.

https://github.com/dotnet/sdk/pull/50662 should fix this issue, but it wasn't in RC1. It should be in RC2.

I added a comment to the code.